packages/styles/scss/components/leadspace/_leadspace.scss

```diff
@@ -246,6 +246,7 @@ $btn-min-width: 26;
 
       ::slotted(#{$dds-prefix}-button-group),
       ::slotted([slot='action']),
+      ::slotted([data-autoid$='button-group']),
```

Comment:
Clever! I think this strikes a nice balance between being fuzzy enough to select the appropriate elements, but specific enough that unwanted elements won't inadvertently pick up the styles.

Reply:
Thank you so much @jkaeser!!!!
I have addressed the other selector in the latest commit. 
No other occurrences were found within the same folder.

Reply:
Agreed - especially the use of `$=` to make sure we don't accidentally select `button-group-item` or similar.

What are your thoughts on adding one more selector - `::slotted([slot="action"].#{$prefix}--buttongroup`. This way items that are slotted there other than `<*-button-group>` have the option to opt-in to the padding rules?